code/datums/mutations/touch.dm

```diff
@@ -49,3 +54,46 @@
 	else
 		to_chat(user,"<span class='warning'>The electricity doesn't seem to affect [target]...</span>")
 		return ..()
+
+#define HALFWAYCRITDEATH ((HEALTH_THRESHOLD_CRIT + HEALTH_THRESHOLD_DEAD) * 0.5)
+
+/obj/item/melee/touch_attack/shock/proc/defib(mob/living/carbon/target, mob/living/carbon/user)
+
+	if(target.can_defib() == DEFIB_POSSIBLE)
+		target.notify_ghost_cloning("Your heart is being defibrillated!")
+		target.grab_ghost() // Shove them back in their body.
+
+	user.visible_message("<span class='warning'>[user] begins to place their hand on [target]'s chest.</span>", "<span class='warning'>You begin to place your hand on [target]'s chest...</span>")
+	if(do_after(user, 5 SECONDS, target))
```

Comment:
isn't the lore of this mutation basically zapping someone with static electricity? shouldn't this be, like, instant?